posthog/api/person.py

```diff
@@ -275,6 +276,12 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.
             else None
         )
 
+        # TEMPORARY: Work out usage patterns of this endpoint
+        statsd.incr("api_person_list_total", tags={"team_id": team.pk})
```

Comment:
we use prometheus for all new ones, this one will only work for US, not EU.

Reply:
Fixed!